.github/workflows/std.yml

```diff
@@ -67,25 +63,36 @@ jobs:
     name: ${{ matrix.target.jobName }}
     runs-on: ubuntu-latest
     needs: discover
-    if: fromJSON(needs.discover.outputs.hits).oci-images.publish != '{}'
+    env:
+      AWS_REGION: us-east-1
+      AWS_ROLE_ARN: arn:aws:iam::926093910549:role/lace-ci
+      # NIX_UPLOAD_CACHE: s3://lace-nix-cache?region=us-east-1
+    permissions:
+      id-token: write
+      contents: read
     strategy:
       matrix:
-        target: ${{ fromJSON(needs.discover.outputs.hits).oci-images.publish }}
+        target: ${{ contains(fromJSON(needs.discover.outputs.hits), 'oci-images') && fromJSON(needs.discover.outputs.hits).oci-images.publish || fromJSON('["dummy-target"]') }}
     steps:
       - name: Configure AWS Credentials
+        if: matrix.target != 'dummy-target'
```

Comment:
Can we make do without dummy target?

Reply:
I tried a few times, but failed. We need this `images` matrix job to succeed, whether it builds anything or not. GitHub doesn’t consider empty matrix jobs as successful (it doesn’t even allow that when evaluating the YAML). If you add an `if:` to the job, it won’t run if the matrix was to be empty, and then our diff and deployment won't run either. So this ugly thing is the only thing I found working… WDYT?

Reply:
Cant we have an if statement like this for whole job, as in just don't run the job if there are no tagets

Reply:
Yes, we can, but then, as I wrote, we won’t run the deployment:

> if the [build] matrix was to be empty, […] then our diff and deployment won't run either